docs/EVM/evm_debug_api.md

```diff
@@ -0,0 +1,76 @@
+
+* debug_traceTransaction - requires the hash of the transaction to be traced, optional parameters:
+  - `disableStorage(boolean)` - (default: false) setting this to true disables storage capture
+  - `disableMemory(boolean)` - (default: false) setting this to true disables memory capture
+  - `disableStack(boolean)` - (default: false) setting this to true disables stack capture
+* [trace_filter](https://openethereum.github.io/JSONRPC-trace-module#trace_filter) - optional parameters:
+  - `fromBlock(uint blockNumber)` - either block number (hex), earliest which is the genesis block or latest (default) best block available. Trace starting block
+  - `toBlock(uint blockNumber)` - either block number (hex), earliest which is the genesis block or latest best block available. Trace ending block
+  - `fromAddress(array addresses)` - filter transactions done from these addresses only. If an empty array is provided, no filtering is done with this field
+  - `toAddress(array addresses)` - filter transactions done from these addresses only. If an empty array is provided, no filtering is done with this field
+  - `after(uint offset)` - default offset is 0. Trace offset (or starting) number
+  - `count(uint numberOfTraces)` - number of traces to display in a batch
+
+There are a couple default values that you should be aware of:
+
+* The maximum number of trace entries a single request of `trace_filter` is allowed to return is `500`. A request exceeding this limit will return an error
+* Blocks processed by requests are temporarily stored on cache for `300` seconds, after which they are deleted.
+
+To change the default values you can add CLI flags when spinning up your tracing node.
+
+## Run a debug node
+
+To use the supported RPC methods, you need to run a node in debug mode, which is slightly different than running a full node. Additional flags will also need to be used to tell the node which of the non-standard features to support.
+
+Spinning up a debug or tracing node is similar to running a full node. However, there are some additional flags that you may want to enable specific tracing features:
+
+* `--ethapi=debug` - optional flag that enables `debug_traceTransaction`
+* `--ethapi=trace` - optional flag that enables `trace_filter`
+* `--wasm-runtime-overrides=<path/to/overrides>` - required flag for tracing that specifies the path where the local Wasm runtimes are stored
+* `--runtime-cache-size 64` - required flag that configures the number of different runtime versions preserved in the in-memory cache to 64
+* `--ethapi-trace-max-count <uint>` - sets the maximum number of trace entries to be returned by the node. _The default maximum number of trace entries a single request of trace_filter returns is_ **500**
+* `--ethapi-trace-cache-duration <uint>` - sets the duration (in seconds) after which the cache of `trace_filter`, for a given block, is discarded. _The default amount of time blocks are stored in the cache is **300** seconds_
+
```

Comment:
How about adding a few command line examples?
Perhaps one that works with a local testnode?

Having something that user can _copy/paste_ and start playing around with immediately is always helpful.

Reply:
Added launch command for Shibuya tracing node.